docs/app/layouts.md

```diff
@@ -0,0 +1,273 @@
+# Layouts
+
+> The Content Module provides Layouts, which are customized displays for viewing and interacting with Items on a
+> Collection Page. Whether your Collection contains blog posts, project management tasks, geo-coordinates, temporal data
+> or beyond; Layouts display Items in a logical and intuitive way.
```

Comment:
Man, I think for a no-code user it might make sense to have the fluff.
The thing is a no-code user it won't occur that "oh, since there are layouts, and they are modular and OS, I can make a layout for *anything*." ...... In stead they will go "hmm, let's see what Layout examples are on-screen".

But I can cut it down if you confirm you want that @rijkvanzanten

Reply:
> Layouts are customized displays for viewing and interacting with the Items in a Collection. Layouts are designed make working with some general category of data model, such as map locations or calendar events, a more human-friendly experience.

Reply:
How's that?

Reply:
bueno